app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision_workflow/general.rb

```diff
@@ -6,7 +6,11 @@ module ManageIQ::Providers::IbmCloud::VPC::CloudManager::ProvisionWorkflow::Gene
   # @param _options [void]
   # @return [Hash] Hash with ems_ref as key and name as value.
   def provision_type_to_profile(_options = {})
-    @provision_type_to_profile ||= index_dropdown(ar_ems.flavors)
+    return {} if ar_ems.nil?
+
+    @provision_type_to_profile ||= begin
+      index_dropdown(ar_ems.flavors)
+    end
```

Comment:
If these are one line now this can stay as `@provision_type_to_profile ||= index_dropdown(ar_ems.flavors)`